w03/d2/morning_exercise/index.html

```diff
@@ -2,12 +2,21 @@
 <html lang="en" dir="ltr">
   <head>
     <meta charset="utf-8">
+<<<<<<< HEAD
```

Comment:
Always remember to clean these up before you `push` code to github. This can cause issues for your teammates.